tests/console/yast2_ftp.pm

```diff
@@ -46,8 +46,8 @@ sub vsftd_setup_checker {
 sub vsftpd_firewall_checker {
     die "service configuration file is missing in firewalld" if script_run("[[ -e /usr/lib/firewalld/services/vsftpd.xml ]]");
     die "vsftpd is not enabled in firewalld"                 if script_run("firewall-cmd --list-services | grep vsftpd");
-    die "Port 21 is not opened"                              if script_run('iptables -L -n -v | grep "tcp dpt:21"');
-    die "Ports for passive ftp are not opened"               if script_run('iptables -L -n -v | grep "tcp dpts:30000:30100"');
+    die "ftp port is not configured"                         if script_run("firewall-cmd --permanent --service ftp --get-ports");
```

Comment:
As we check this service, should not we also check that it's enabled,  like we do on L48 for vsftpd?

Side note, you have said that firewalld cli cannot list open ports, which can be easily done by `firewall-cmd --list-ports`. But as we agreed, it's out of scope for testing yast module, as yast just enabled the service.

Reply:
I have tried that, unfortunately it doesn't work as one would expect.
https://unix.stackexchange.com/questions/422242/why-firewall-cmd-cant-list-active-ports-on-my-local-pc

Reply:
That's alright, as we don't have active connections and it was a side note. What's about first part of the comment?

Reply:
I added "systemctl is-enabled vsftpd" . Is that what you had in mind?
http://falafel.suse.cz/tests/982#step/yast2_ftp/129

Reply:
Not really, so for vsftpd we have this check `firewall-cmd --list-services | grep vsftpd`, but then we also check ports of the `ftp`  service. So in case YaST enables both (which I guess is not the case), we should also check that ftp service is added. If not, I'm not sure why we want to list ports.

Reply:
Ok, so indeed once I configure vsftpd , port 21 is opened but ftp service is not used. I will remove the check of ftp service port, once my workstation is up again. I can, instead check the usr/lib/services/firewalld/vsftpd.xml where the tcp post should be set to 21.
`<?xml version="1.0" encoding="utf-8"?>
<service>
  <short>vsftpd ports</short>
  <description>vsftpd ports for both active and passive connections</description>
  <port protocol="tcp" port="21"/> <!-- control connection port -->
  <port protocol="tcp" port="30000-30100"/> <!-- passive mode port range -->
</service>
`
WDYT?

Reply:
I would say that we rely on vsftpd tests to verify that valid config is provided by the package, so let's remove the check for ftp service completely. Thanks!

Reply:
Done